core/tests/test_utils.py

```diff
@@ -197,6 +199,206 @@ def get_storage_model_classes():
                     yield clazz
 
 
+class ElasticSearchServicesStub(python_utils.OBJECT):
```

Comment:
As per offline discussion, in the future, it would be better to create stub of the third-party elasticsearch.Elasticsearch class OR use ElasticMock lib if possible.

Is that correct? (Just wanted to have a link and have confirmation before creating an issue for this.)